cmd/main.go

```diff
@@ -482,6 +482,8 @@ func main() {
 			} else {
 				failOnError(uploadISOs(objectHandler, versionHandler, log), "Failed to upload boot files")
 			}
+		} else {
+			log.Infof("Skipping ISO upload, image service running at %s", bm.ImageServiceBaseURL)
```

Comment:
Does this mean the assisted-service will enable the API and report ready regardless of the image-service status and the availability of the images?

Reply:
OK, I see you adressed it in `wait_for_assisted_service.py`
Yet we might want to reflect the service isn't really functional as long the image-service images aren't avilable

Reply:
I mean it is to a degree. If you wanted to, you could create an infra-env then the request to download the image would just fail.

The assisted service doesn't directly depend on the image service (i.e. nothing will fail in the assisted service because the image service isn't up).